netket/hilbert/__init__.py

```diff
@@ -15,6 +15,8 @@
 from .abstract_hilbert import AbstractHilbert
 from .discrete_hilbert import DiscreteHilbert
 
+from .continuous_hilbert import Particles
```

Comment:
I'd stop calling things in `/hilbert` with `_hilbert` because that's obvious (of course it is our bad).
I'd call this file `continuous_particles` or something like that instead of `continuous_hilbert` (we might have a continuoushilbert later on, and it's not particles...)